src/extension/extension.ts

```diff
@@ -449,10 +452,19 @@ export function activate(context: vscode.ExtensionContext): void {
     const jsonContent = JSON.parse(strContent)
     if (jsonContent && jsonContent.build_only) {
       delete jsonContent.build_only
-      const newConfFileUri = getConfUri(
-        jsonContent.verify[0].split(':')[0] +
-          getFileName(file.path).replace('last_conf', ''),
-      )
+      let verifyStr = ''
+      if (typeof jsonContent.verify === 'string') {
+        verifyStr = jsonContent.verify
+      } else {
```

Comment:
this is for the title of a conf that was converted from shell script
right now, the title for this kind of converted conf is the name of the main contract + date&time
so I just need the string value of "verify", if it exists.
because it can be either a string or an array with 1 string, I first check if it is a string, then if it doesn't, I assume it is an array with 1 string
I can add a check for an array, I'm not sure it is necessary though